DoSomethingArchive/legacy-website
# Sends Reportback from Phoenix API -> Rogue

#### What's this PR do?
- When an app hits Phoenix's `/api/v1/campaigns/[nid]/reportback` endpoint, instead of saving directly to Phoenix, it will send along to Rogue first and then Rogue will send back to Phoenix to be saved.
- If crop values aren't set, don't send to Rogue (both on first send and on cron job).

#### How should this be reviewed?
- Hit the `/api/v1/campaigns/[nid]/reportback` endpoint.
![screen shot 2016-11-17 at 3 51 11 pm](https://cloud.githubusercontent.com/assets/9019452/20407050/b646eb72-acdd-11e6-837f-9aa246408c0b.png)
- Check the Rogue `reportback_items` table for the reportback item.
- Check the Phoenix `dosomething_reportback_file` table for the reportback item. 
- Check that Phoenix's `dosoemthing_rogue_reportbacks` table has the reportback item with all correct data.

#### Relevant tickets
Fixes #7183 

#### Checklist
